Marlin/Configuration.h

```diff
@@ -2581,6 +2581,7 @@
  * you must uncomment the following option or it won't work.
  */
 //#define SDSUPPORT
+//#define SDIO_SUPPORT
```

Comment:
The pins files automatically define `SDIO_SUPPORT` when SDIO is available and possible (e.g., SDIO pins are not used for something else). Most don't currently allow for disabling it (by setting `SDIO_SUPPORT` to `false`) but that could be made possible throughout the pins files that define it.